perl/INSTALL.md

```diff
@@ -0,0 +1,107 @@
+# Bystro Annotator Package Installation
+
+## Installing Bystro Annotator using Docker
+
+To build a Docker image using the `Dockerfile`, run the following:
+
+```bash
+cd ../ && docker build -t bystro-annotator -f Dockerfile.perl .
```

Comment:
Am I supposed to start in the bystro repo folder to run this, or does it intend for me to start in the bystro/perl folder so it takes me to the bystro folder?